core/workspace.js

```diff
@@ -178,6 +178,20 @@ Blockly.Workspace.prototype.getAllBlocks = function() {
 };
 
 /**
+ * Checks whether all inputs are filled.
```

Comment:
/**
 * Checks whether all value and statement inputs in the workspace are filled with blocks.
 * Shadow blocks count as filling an input.
 * @return {boolean} True if all inputs are filled, false otherwise.
 */